src/data_provider/src/sysInfoLinux.cpp

```diff
@@ -242,22 +260,28 @@ static nlohmann::json parseRpm(const std::string& packageInfo)
     auto it{info.find("Name")};
     if (it != info.end() && it->second != "gpg-pubkey")
     {
-        std::string version;
         ret["name"] = it->second;
+
+        std::string size         { UNKNOWN_VALUE };
+        std::string install_time { UNKNOWN_VALUE };
+        std::string groups       { UNKNOWN_VALUE };
+        std::string version;
```

Comment:
did you forget this?

Reply:
Nope. I kept this empty because "version" is used to be appended some lines below.

Reply:
Fixed in last commit :)